### PR TITLE
feature: remove InstanceInfo from container env

### PR DESCRIPTION
**Pull request by allencloud**
Signed-off-by: Allen Sun <allensun.shl@alibaba-inc.com>

### Ⅰ. Describe what this PR did

remove InstanceInfo from container env since this part could be added in the plugin.
And this block of code is for special use, not so general.

@Ace-Tang 

### Ⅱ. Does this pull request fix one issue?

none

### Ⅲ. Describe how you did it
none

### Ⅳ. Describe how to verify it
none

### Ⅴ. Special notes for reviews
none

### CODE REVIEW

File: daemon/mgr/container_envs.go [hunk 1]
```diff
@@ -25,33 +24,24 @@ import (
...
 )
 
 // updateContainerEnv update the container's envs in
-// /etc/instanceInfo and /etc/profile.d/pouchenv.sh
-// Env used by rich container.
+// /etc/profile.d/pouchenv.sh used by rich container.
 func updateContainerEnv(inputRawEnv []string, baseFs string) error {
 	var (
-		envPropertiesPath = path.Join(baseFs, containerInstanceInfo)
-		envShPath         = path.Join(baseFs, pouchEnvFile)
```
Comment (Ace-Tang) [inline daemon/mgr/container_envs.go]: Where do we create `envShPath` and `envShDir`, are they create in rich-mode with runc?

Comment (allencloud) [reply daemon/mgr/container_envs.go]: The file is located in the image's rootfs. 
In addition, the file is in the base image.

Comment (Ace-Tang) [reply daemon/mgr/container_envs.go]: do you mean we not create the file, but only judge the file

Comment (allencloud) [reply daemon/mgr/container_envs.go]: yes

File: daemon/mgr/container_envs.go [hunk 2]
```diff
@@ -25,33 +24,24 @@ import (
...
 	// if dir of pouch.sh is not exist, it's unnecessary to update that files.
 	if _, err := os.Stat(path.Join(baseFs, pouchEnvDir)); err != nil {
 		return nil
 	}
-	if _, err := os.Stat(envPropertiesPath); err != nil {
-		//if etc/instanceInfo is not exist, it's unnecessary to update that file.
-		return nil
-	}
 	if _, err := os.Stat(envShPath); err != nil {
 		return fmt.Errorf("failed to stat container's env file /etc/profile.d/pouchenv.sh: %v", err)
```
Comment (Ace-Tang) [inline daemon/mgr/container_envs.go]: if `envShPath` is not exist, we should return nil, as you said, we just judge the file i exist.

Comment (allencloud) [reply daemon/mgr/container_envs.go]: Using
```
	if _, err := os.Stat(envShPath); err != nil {
		logrus.Warnf("failed to stat container's env file /etc/profile.d/pouchenv.sh: %v", err)
		return nil
	}
```
? @Ace-Tang

**Comment by Ace-Tang** [comment]
LGTM

**Pull request merged**